README.md

```diff
@@ -31,16 +31,44 @@ Clipboard Utilities
  - Windows - clip
  - Linux   - xsel
 
+Commands
+--------
+
+System copy adds vim commands for copying `:SystemCopy`,
+`:SystemCopyLine` as well as for pasting `:SystemPaste`.
+
+Options
+-------
+
+System copy allows user to specify the custom command for copy and paste
+functions.
+
+To declare custom copy command use following example:
+``` vim
+let g:system_copy#copy_command='xclip -sel clipboard'
+```
+And to declare custom paste command use:
+``` vim
+let g:system_copy#paste_command='xclip -sel clipboard -o'
+```
```

Comment:
Nice! I like the flexibility, while still using the basic configuration as needed.